samples/api-reference-doc-snippets/instance.js

```diff
@@ -362,6 +362,97 @@ const snippets = {
       });
     // [END bigtable_api_del_instance]
   },
+
+  executeQuery: (instanceId, tableId) => {
+    // [START bigtable_api_execute_query]
+    const {Bigtable} = require('@google-cloud/bigtable');
+    const bigtable = new Bigtable();
+    const instance = bigtable.instance(instanceId);
+
+    const query = `SELECT
+                    _key
+                  from \`${tableId}\` WHERE _key=@row_key`;
+    const parameters = {
+      row_key: 'alincoln',
+    };
+
+    const parameterTypes = {
+      row_key: Bigtable.ExecuteQueryTypes.String(),
```

Comment:
This needs to be `SqlTypes` now and it needs to be a separate import, it's not namespaced under Bigtable

Reply:
Done.